### PR TITLE
Correction of tips (content, trigger, ...)

### PR DESCRIPTION
**Pull request by qsm-odoo**
PAD : https://pad.odoo.com/p/r.JokowwPZJRXyfeZp
TASK : https://www.odoo.com/web#id=11037&view_type=form&model=project.task&menu_id=3942&action=327

### CODE REVIEW

File: addons/account/account_tip_data.xml [hunk 1]
```diff
@@ -3,25 +3,21 @@
...
             <field name="description">Click here to generate a PDF file of your financial reports (like Balance Sheet, Profit <![CDATA[&amp;]]> Loss, Ledgers, etc...)</field>
-            <field name="action_id" ref="account.action_bank_statement_tree"/>
             <field name="model">account.bank.statement</field>
-            <field name="type">form</field>
             <field name="mode">form</field>
-            <field name="trigger_selector">header button span:contains("Close"):visible</field>
-            <field name="highlight_selector">span.oe_menu_text:contains('Legal Reports')</field>
+            <field name="trigger_selector">header button#close_statement:visible</field>
+            <field name="highlight_selector">.oe_secondary_menus_container a[data-menu='227']</field>
             <field name="placement">right</field>
```
Comment (vme-odoo) [inline addons/account/account_tip_data.xml]: Why static data-menu *227*, is that specific for all databases ??

File: addons/account_bank_statement_import/account_import_tip_data.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
             <field name="description">Click here to import your electronic bank statements. This will import all the movements in and out across all your accounts.</field>
             <field name="action_id" ref="account.action_invoice_tree1"/>
             <field name="model">account.invoice</field>
             <field name="mode">form</field>
-            <field name="trigger_selector">header button.oe_button:nth-child(8):not(.oe_form_invisible):has(span:contains('Register Payment')), .oe_secondary_menus_container .oe_secondary_menu ul li span.oe_menu_text:contains('Bank Statements'):not(:first):visible</field>
-            <field name="highlight_selector">.oe_secondary_menus_container .oe_secondary_menu ul li span.oe_menu_text:contains('Bank Statements'):not(:first)</field>
+            <field name="trigger_selector">header #register_payment:not(.oe_form_invisible),.oe_secondary_menus_container .oe_secondary_menu ul li a.oe_menu_toggler[data-menu='222']</field>
+            <field name="highlight_selector">.oe_secondary_menus_container .oe_secondary_menu ul li a.oe_menu_toggler[data-menu='222']</field>
```
Comment (vme-odoo) [inline addons/account_bank_statement_import/account_import_tip_data.xml]: same as above.

**Comment by qsm-odoo** [comment]
Yes, after reflexion these ids must be specific to db. I will find a way to fix this.

**Comment by qsm-odoo** [comment]
New pull request here : https://github.com/odoo/odoo/pull/4501

**Pull request closed without merging**